spec/parser/ruby/ruby_parser_spec.rb

```diff
@@ -283,6 +283,54 @@ def hello; end
       end
     end
 
+    it "correcly parses several formats of percent arrays in constant declarations" do
+      delimiter_a = [
+        ['(', ')'],
+        ['{', '}'],
+        ['[', ']'],
+        ['<', '>'],
+        ['|', '|']
+      ]
+
+      type_a = [
```

Comment:
This may as well be a hash for readability (since order doesn't matter for 1.8 anyway)